lib/LicenseTermsList/LicenseTermsList.js

```diff
@@ -35,24 +38,32 @@ export default class LicenseTermsList extends React.Component {
     const note = get(this.props.license, ['customProperties', term.name, '0', 'note']);
 
     return (
-      <Layout className="padding-top-gutter" key={index}>
-        <Row>
-          <Col xs={12} md={4}>
-            {term.description ? <InfoPopover data-test-info-popover content={term.description} /> : null}
-            <strong data-test-term-label={term.name}>{term.label}</strong>
-          </Col>
-          <Col xs={12} md={8}>
-            <span data-test-term-value={term.name}>{value}</span>
-          </Col>
-        </Row>
-        { note ? (
+      <Layout key={index}>
+        <Card
+          id="term-card"
```

Comment:
This will create multiple items with the same `id` and `id`s should be unique across an entire page. Consider appending the `index` to the `id` like:
```suggestion
          id={`term-card-${index}`}
```